lib/routes/sms.js

```diff
@@ -98,6 +101,53 @@ module.exports = (log, isA, error, config, customs, sms) => {
           return {}
         }
       }
+    },
+    {
+      method: 'GET',
+      path: '/sms/status',
+      config: {
+        auth: {
+          strategy: 'sessionToken'
+        }
+      },
+      handler (request, reply) {
+        log.begin('sms.status', request)
+
+        let isOk
+
+        return getStatus()
```

Comment:
I realize there are only a couple of function calls, but I found the `isOk` check in `getLocation` a bit confusing before my brain has fully woken up. An alternative that might be easier to follow is to use Promise.all then array destructuring or [Promise.spread](http://bluebirdjs.com/docs/api/spread.html) on the result:

```js
return Promise.all([
  getStatus(),
  getLocation()
])
.spread(createResponse)
.then(reply, reply);

function getStatus () {
  return sms.status()
    .then(statuts => status.isOk)
    .catch(...);
}

function getLocation () {
  return getGeoData(request.app.clientAddress)
    .then(result => REGIONS.test(result.location.countryCode))
    .catch(...);
}

function createResponse(isStatusOk, isLocationOk) {
  return { ok: isStatusOk && isLocationOk };
}
```